openedx/core/djangoapps/config_model_utils/admin.py

```diff
@@ -28,9 +29,48 @@ class StackedConfigModelAdmin(ConfigurationModelAdmin):
     """
     form = StackedConfigModelAdminForm
 
-    def get_fields(self, request, obj=None):
+    raw_id_fields = ('course',)
+
+    def get_fieldsets(self, request, obj=None):
+        return (
+            ('Context', {
+                'fields': self.key_fields,
+                'description': _(
+                    'These define the context to enable this configuration on. '
+                    'If no values are set, then the configuration applies globally. '
+                    'If a single value is set, then the configuration applies to all courses '
+                    'within that context. At most one value can be set at a time.<br>'
+                    'If multiple contexts apply to a course (for example, if configuration '
+                    'is specified for the course specifically, and for the org that the course '
+                    'is in, then the more specific context overrides the more general context.'
+                ),
+            }),
+            ('Configuration', {
+                'fields': self.stackable_fields,
+                'description': _(
+                    'If any of these values is left empty or "Unknown", then their value '
```

Comment:
nit: I think it should be either "values are left empty" or "then its value"

Reply:
Agreed.